src/Data/Report.cs

```diff
@@ -19,10 +19,8 @@ public class Report
 
         public string ToolVersion { get; init; }
 
-        [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault | JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
This was a bit weird from the start, given for a string, null is the default value.